op-mode-definitions/poweroff.xml

```diff
@@ -2,37 +2,45 @@
 <interfaceDefinition>
   <node name="poweroff">
     <properties>
-      <help>Poweroff the system</help>
+      <help>Shutdown the system</help>
     </properties>
-    <command>/opt/vyatta/bin/sudo-users/vyatta-poweroff.pl --action poweroff</command>
+    <command>sudo ${vyos_op_scripts_dir}/powerctrl.py --poweroff now</command>
+
     <children>
       <leafNode name="now">
         <properties>
-          <help>Poweroff the system without confirmation</help>
+          <help>Shutdown the system imidiently</help>
```

Comment:
Typo `immediately` instead of `imidiently`. Also see comment above, I prefer the explicit wording of `without confirmation`, instead of the vague term `immediately`.

Reply:
undoing